board_types.txt

```diff
@@ -206,6 +209,11 @@ AP_HW_CUBEBLACK_PERIPH               1401
 AP_HW_PIXRACER_PERIPH                1402
 AP_HW_SWBOOMBOARD_PERIPH             1403
 
+#IDs 5600-5699 reserved for ZeroOne
+AP_HW_ZeroOne_X6                     5600
```

Comment:
is AP_HW_ZeroOne_X6 and  ZEROONE_X6 the same board? If so since you already have the AP id in Ardupilot, just add it here. Do not "Create" Ardupilot ID here that do not exist in Ardupilot yet.

Reply:
Yes, this line already exists in the Ardupilot.